wordpress-mobile/WordPress-iOS
# Stats: Scroll to new Insights card when added

This PR adds functionality so that when a user adds a new Insights card, the screen will scroll to the position of their new Insights card

Ref #17846

To test:
1. Go to Stats screen and then click Insights tab
2. Cog (gear) icon should appear in top right
3. Tap the cog (gear) icon and add a new stats card.  Screen should scroll to the position of the new stats card

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:
